hackvote/app/models/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event < ActiveRecord::Base
+  has_many :projects
+  # order so we can display past hackdays
+  default_scope -> { order(created_at: :desc) }
+  # make sure we have title for event, and title doesn't overflow string.
+  validates :title, presence: true, length: { maximum: 255 }
+
+  def project_feed
+  	Project.where("event_id = ?", id)
```

Comment:
Yep, the has_many line basically just adds a bunch of methods to let you access the association: http://guides.rubyonrails.org/association_basics.html#has-many-association-reference